docs/hosting/configuration/environment-variables/external-hooks.md

```diff
@@ -18,3 +18,4 @@ You can define external hooks that n8n executes whenever a specific operation ru
 | Variable | Type  | Description |
 | :------- | :---- | :---------- |
 | `EXTERNAL_HOOK_FILES` | String | Files containing external hooks. Provide multiple files as a colon-separated list ("`:`"). |
+| `EXTERNAL_FRONTEND_HOOKS_URLS` | String | URLs to external frontend hooks files. Provide multiple files as a colon-separated list ("`:`"). |
```

Comment:
Might want to rephrase this. I'm guessing the files aren't external - it's the hooks that are external?

Reply:
Made some changes in the [latest commit](https://github.com/n8n-io/n8n-docs/pull/2147/commits/a04501518f7eeaad3f26f48ccf0aa61ee2ac8c11)